src/kbc_modules/cc_kbc/crypto/mod.rs

```diff
@@ -11,6 +12,7 @@ use rsa::{PaddingScheme, RsaPrivateKey, RsaPublicKey};
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha384};
 
+const RSA_KEY_TYPE: &str = "RSA";
 const RSA_ALGORITHM: &str = "rsa-pkcs1v15";
```

Comment:
Here I think we should use`RSA1_5` follow Section 4.1 of [rfc7518](https://www.rfc-editor.org/rfc/inline-errata/rfc7518.html), and so as bellow (`AES_GCM_256_ALGORITHM`)

Reply:
We may should also change `const AES_GCM_256_ALGORITHM: &str = "aes-gcm-256";` to `const AES_GCM_256_ALGORITHM: &str = "A256GCM"` to follow Section 5.3 of [rfc7518](https://www.rfc-editor.org/rfc/inline-errata/rfc7518.html).